Networking/Sources/Extended/Model/Product/ProductAttribute.swift

```diff
@@ -93,6 +93,23 @@ public extension ProductAttribute {
     var isGlobal: Bool {
         !isLocal
     }
+
+    /// This is intended for use for converting variations-as-products to variations.
+    /// This inverts the assumption made when we decoded `option` as `options = [option]`
+    ///
+    func toProductVariationAttribute() throws -> ProductVariationAttribute {
```

Comment:
nit: do we plan to have other use cases of this conversion? if not, this extension could be private to the use case in `POSProductOrVariationResolver`.

Reply:
I had it like that originally. However, with the strange `option`/`options` decoding logic, I think it makes the code clearer to keep it all together.